.github/workflows/build.yml

```diff
@@ -10,6 +10,7 @@ jobs:
     strategy:
       matrix:
         arduino-platform: ["esp32:esp32"]
+        sketches: ["TIOL11X_master"]
```

Comment:
I will eventually implement board definitions into the library.  I know the ESP32S3 has up to 3 UART ports.  Will need to cap the number of ports and possibly standardize some of the pin definitions.  Will probably create an issue around this.